articles/openshift/oshift-ref-no_proxy.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Specifying endpoints to avoid being proxied in deployments with government community network connectivity | UKCloud Ltd
+description: Provides information regarding proxy environment variables in v3.11 OpenShift clusters deployed with government community network connectivity.
+services: openshift
+author: Ben Bacon
+toc_rootlink: Reference
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Specifying endpoints to avoid being proxied in deployments with government community network connectivity
+toc_fullpath: Reference/oshift-ref-no_proxy.md
+toc_mdlink: oshift-ref-no_proxy.md
+---
+
+# Specifying endpoints to avoid being proxied in deployments with government community network connectivity
+
+## Overview
+
+This article aims to describe the purpose of the `no_proxy` variable in OpenShift deployments with connectivity to government community networks. By default, where containers have the relevant proxy environment variables set, all outbound HTTP(S) requests will be initiated via the proxy (which only has internet connectivity). Adding entries to the `no_proxy` variable enables outbound traffic destined for specific hosts or CIDRs to bypass Squid proxy on nodes with access to alternative networks.
+
+### Intended audience
+
+To follow the steps in this article you must have access to a cluster that is running OpenShift Container Platform v3.11 (or newer) with connectivity to government community networks (such as HSCN, PSN or Janet). Within this cluster configuration we deploy Squid proxy to enable non-internet connected nodes to request internet resources and data in a restricted manner.
+
+## Squid
+
+Squid proxy is installed as a service on the control plane load balancers and enables containers running on worker nodes (with government community network access) to request resources from the internet. Our default configuration only permits access to specific endpoints to ensure compliance with network regulatory requirements. You can find more information regarding these endpoints in [*How to permit outbound access to internet hosts in deployments with government community network connectivity*](oshift-how-add-domains-proxy-whitelist.md).
+
+This article also provides steps for how you can add your own domains from within OpenShift.
```

Comment:
Does this belong in this section or would it be better move it to the end of the overview section? If it's not directly related to Squid, move it.